cfg/conf.ini.sample

```diff
@@ -3,9 +3,6 @@
 ; An explanation of each setting can be find online at https://github.com/PrivateBin/PrivateBin/wiki/Configuration.
 
 [main]
-; (optional) set a project name to be displayed on the website
-; name = "PrivateBin"
-
```

Comment:
Please merge the current master into your project. You've accidentally reverted some commits/code changes we did in the master branch recently.